public/course_wizard.php

```diff
@@ -0,0 +1,261 @@
+<!--This works for now but should probably be refactored into separate html/php files.
+It was mostly to try mixed php/html and get something working with basic styling
+and data entry. -->
+
+<html>
+
+    <?php
+
+    require "../modules/models/course.php";
+    require "../modules/models/course_offering.php";
+    require "../modules/models/course_section.php";
+    require_once "../common.php";
+
+    try  {
+        $result_courses = Course::getAll();
+        $result_offerings = CourseOffering::getAll();
+        $result_sections = CourseSection::getAll();
+    } catch(PDOException $error) {
+        echo $sql . "<br>" . $error->getMessage();
+    }?>
+
+<!-- This section deals with submission of the entered data.-->
+
+    <?php if (isset($_POST['submitCourse'] ))
+    {
+        $course = new Course();
+        $course->course_code = $_POST['course_code'];
+        $course->course_name = $_POST['course_name'];
+        
+        try{
+            $course->save();
+            $create_success = true;
+        }catch(PDOException $error) {
+            echo $sql . "<br>" . $error->getMessage();
+        }
+    }
+
+    elseif(isset($_POST["submitOffering"]))
+    {
+        $offering = new CourseOffering();
+        $offering->course_id = $_POST['course_selection'];
+        $offering->course_offering_code = $_POST['course_offering_code'];
+        $offering->course_offering_name = $_POST['course_offering_name'];
+        
+        try{
+            $offering->save();
+            $create_success = true;
+        }catch(PDOException $error) {
+            echo $sql . "<br>" . $error->getMessage();
+        }
+    }
+    elseif(isset($_POST["submitSection"]))
+    {
+        $section = new CourseSection();
+        $section->offering_id = $_POST['offering_selection'];
+        $section->course_section_code = $_POST['course_section_code'];
+        $section->course_section_name = $_POST['course_section_name'];
+        
+        try{
+            $section->save();
+            $create_success = true;
+        }catch(PDOException $error) {
+            echo $sql . "<br>" . $error->getMessage();
+        }
+    }
+    ?>
+
+    <!-- Seemed like a good way to prevent form resubmission on refresh? -->
+    <?php if ((isset($_POST['submitCourse']) ||isset($_POST['submitOffering']) ||isset($_POST['submitSection'])) && $create_success)
```

Comment:
$create_success could potentially be undefined here

Reply:
Thanks! Corrected by adding an isset() check prior.